app/main.py

```diff
@@ -52,6 +52,10 @@ def root(request: Request):
 async def favicon():
     """
     Overrides the default favicon with a custom one.
+
+    NOTE: When the API is behind an NGINX reverse proxy that has a stripped path prefix (and root_path is defined),
```

Comment:
should we just say "reverse proxy" now, instead of NGINX?

Reply:
also: root_path -> base_path

Reply:
I use `root_path` to refer to the variable [defined in FastAPI](https://fastapi.tiangolo.com/advanced/behind-a-proxy/#providing-the-root_path) for a base path :)